src/main/resources/jenkins/plugins/hipchat/workflow/HipChatStep/config.jelly

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:entry field="message" title="Message">
+        <f:textbox/>
+    </f:entry>
+    <f:entry field="color" title="Color">
```

Comment:
Not sure how to get an enum to auto-populate for the Workflow Snippet Generator. Anyone have any examples of using an enum as a Workflow step field?